src/library/scala/jdk/CollectionConverters.scala

```diff
@@ -95,19 +84,12 @@ import scala.collection.convert._
   * again will return the original source object. For example:
   *
   * {{{
-  *   import scala.collection.JavaConverters._
+  *   import scala.jdk.CollectionConverters._
   *
   *   val source = new scala.collection.mutable.ListBuffer[Int]
   *   val target: java.util.List[Int] = source.asJava
   *   val other: scala.collection.mutable.Buffer[Int] = target.asScala
   *   assert(source eq other)
   * }}}
-  *
   */
-object CollectionConverters extends AsJavaConverters with AsScalaConverters {
-
-  /** This object provides extension methods that convert between Scala and Java collections,
-    * see [[CollectionConverters]].
-    */
-  object Ops extends AsJavaExtensions with AsScalaExtensions
```

Comment:
This change is going to break Scala.js again. Can we keep `Ops` as is and deprecated, please? Otherwise I'll have to redo a release sprint for Scala.js 0.6.28 and 1.0.0-M8 for Scala 2.13.0-RC2.

Reply:
Sure, I can do that. Though it's a bit strange to put a deprecated object in a new library addition. I assume it would be only for CollectionConverters? Can we make it `private[scala]`?

Reply:
It's only for `CollectionConverters`. However it would need to be public, as we use it from `java.util`.

Reply:
I assume you mean deprecated in RC2 and can be gone in GA?

Reply:
Well, that would give me a bit of room to breathe, but I'd still have to rush to republish two versions of Scala.js before the GA.

I understand you'd not want that deprecated thing in the final version, though ...

Reply:
OK, let's go for that procedure. Thanks!

Reply:
follow-up tracked: https://github.com/scala/bug/issues/11500

Reply:
Thanks!